src/TextToSpeech.hs

```diff
@@ -51,12 +51,22 @@ espeak_contrib myDir lang tmp txt =
     l = case lang of
             Language s    -> s
 
+say :: Language -> FilePath -> String -> (String, [String])
+say lang tmp txt =
+   ("say", ["-o", tmp, "--data-format=LEF32@8000", "-v", l, txt])
+  where
+   l = case lang of
+           Language "en" -> "Alex"
+	   Language "de" -> "Anna"
```

Comment:
Already changed and commited. My editor vim wasn't aware of Haskell files yet. And it was pretty late last night at GPN ;)